pyserini/search/__main__.py

```diff
@@ -19,23 +19,21 @@
 from pyserini.search.reranker import ClassifierType, PseudoRelevanceClassifierReranker
 
 parser = argparse.ArgumentParser(description='Create a input schema')
-parser.add_argument('-index', type=str, metavar='path to index',
-                    required=True, help='the path to lucene index')
-parser.add_argument('-topics', type=str, metavar='topic_name', required=True,
+parser.add_argument('--index', type=str, metavar='path to index', required=True, help='the path to lucene index')
```

Comment:
Per: https://github.com/lintool/guide/blob/master/coding-style.md#describing-arguments-in-command-line-applications

"Path to Lucene index."
"Name of topics. Available: robust04, robust05, core17, core18."
"Path to output file."
"Use BM25 (default)."
"Use RM3."
...

Initial cap, end with period.